tools/ci_build/github/azure-pipelines/android-arm64-v8a-QNN-crosscompile-ci-pipeline.yml

```diff
@@ -3,10 +3,7 @@ parameters:
 - name: QnnSdk
   displayName: QNN SDK version
   type: string
-  default: qnn-v2.8.0.230223123141_52150
-  values:
-  - qnn-v2.6.0.221227110525_42395
-  - qnn-v2.8.0.230223123141_52150
```

Comment:
Removing these value options allows us to manually specify any QNN SDK version when running the pipeline.